packages/google_maps_flutter/google_maps_flutter_ios/example/ios13/ios/RunnerUITests/RunnerUITests.m

```diff
@@ -0,0 +1,78 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <XCTest/XCTest.h>
+
+static const CGFloat kWaitTime = 15;
```

Comment:
Nit: This should be an `NSTimeInterval` rather than a `CGFloat`, since that conveys unit information.